src/pages/preview/[...slug].tsx

```diff
@@ -0,0 +1,67 @@
+import type { GetStaticProps, GetStaticPropsContext, NextPage } from 'next';
+import { ParsedUrlQuery } from 'querystring';
+import React from 'react';
+import { Crumb } from '../../components/breadcrumbs/types';
+import { ContentItem } from '../../interfaces/content-item';
+import { TertiaryNavItem } from '../../components/tertiary-nav/types';
+import ContentPageTemplate, {
+    determineVideoOrPodcast,
+} from '../../components/main-content-page/content-page-template';
+import { getPreviewContent } from '../../service/get-preview-content';
+
+let pluralize = require('pluralize');
+
+interface ContentPageProps {
+    crumbs: Crumb[];
+    topicSlug: string;
+    topicName: string;
+    contentItem: ContentItem;
+    tertiaryNavItems: TertiaryNavItem[];
+    relatedContent: ContentItem[];
+}
+
+const ContentPage: NextPage<ContentPageProps> = ({
+    crumbs,
+    topicSlug,
+    topicName,
+    contentItem,
+    tertiaryNavItems,
+    relatedContent,
+}) => {
+    return (
+        <ContentPageTemplate
+            crumbs={crumbs}
+            topicSlug={topicSlug}
+            topicName={topicName}
+            contentItem={contentItem}
+            tertiaryNavItems={tertiaryNavItems}
+            relatedContent={relatedContent}
+        />
+    );
+};
+
+export default ContentPage;
+
+export const getServerSideProps = async (context: any) => {
```

Comment:
Awesome 🎉

Reply:
ideally we should just repurpose  [...slug].tsx but next.js is not resolving  to catch all [...slug].tsx , so had to make a new path /preview , may be we can explore later.